x-pack/plugin/downsample/src/main/java/org/elasticsearch/xpack/downsample/TransportDownsampleAction.java

```diff
@@ -234,6 +234,8 @@ protected void masterOperation(
             return;
         }
 
+        final TaskId parentTask = new TaskId(clusterService.localNode().getId(), task.getId());
+        downsamplingInterval = request.getDownsampleConfig().getInterval().toString();
```

Comment:
Nit: I'd move this to the top of the function, since this is updating a member variable and only needs the request arg.

Reply:
done: 0bdbea0f19edbbcec330a516314b29ba30f59dc8